app/src/main/java/com/example/rezepteapp/EntranceFragment.java

```diff
@@ -0,0 +1,64 @@
+package com.example.rezepteapp;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link EntranceFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class EntranceFragment extends Fragment {
+
+    // TODO: Rename parameter arguments, choose names that match
```

Comment:
Könntest du die sachen, die wir nicht nutzen raus nehmen?

Reply:
Kann ich machen, die sind in deinem EditFragment auch noch drinnen. Soll ich die nicht benötigten Zeilen auch mal rausnehmen?

Reply:
ja, mach mal